app/packs/src/apps/mydb/elements/Elements.js

```diff
@@ -30,29 +30,25 @@ export default class Elements extends Component {
 
   render() {
     const { currentElement } = this.state;
-    const showReport = (currentElement || []).type === 'report';
+    const hasCurrentElement = currentElement !== null;
 
-    let md = 12;
-    const overview = !(currentElement);
-    let page = null;
-
-    if (currentElement) {
-      md = 5;
-      page = (
-        <Col md={7} className="small-col">
-          <ElementDetails currentElement={currentElement} />
-        </Col>
-      );
-    }
+    const showReport = currentElement?.type === 'report';
+    const listWidth = hasCurrentElement ? 5 : 12;
 
     return (
-      <div>
-        <Col md={md} className="small-col">
-          <ElementsList overview={overview} showReport={showReport} />
+      <Row>
+        <Col md={listWidth}>
```

Comment:
Col widths in bootstrap-react are set to a certain break point.
In general our layouts or rather the whole app are not responsive.
Therefore I would suggest to always use the xs parameter. It results to column widths beings applied regardless of the current window size.